app/app.go

```diff
@@ -41,127 +23,110 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/version"
-
-	// Cosmos-SDK Modules
-	// https://github.com/cosmos/cosmos-sdk/tree/master/x
-	// NB: Osmosis uses a fork of the cosmos-sdk which can be found at: https://github.com/osmosis-labs/cosmos-sdk
-
-	// Auth: Authentication of accounts and transactions for Cosmos SDK applications.
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 	authrest "github.com/cosmos/cosmos-sdk/x/auth/client/rest"
 	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
-
-	// Capability: allows developers to atomically define what a module can and cannot do
 	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
-
-	// Crisis: Halting the blockchain under certain circumstances (e.g. if an invariant is broken).
 	"github.com/cosmos/cosmos-sdk/x/crisis"
-
-	// Evidence handling for double signing, misbehaviour, etc.
-
-	// Params: Parameters that are always available
 	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
-
-	// Upgrade:  Software upgrades handling and coordination.
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-
-	// IBC Transfer: Defines the "transfer" IBC port
 	transfer "github.com/cosmos/ibc-go/v2/modules/apps/transfer"
+	"github.com/gorilla/mux"
+	"github.com/rakyll/statik/fs"
+	"github.com/spf13/cast"
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	"github.com/tendermint/tendermint/libs/log"
+	tmos "github.com/tendermint/tendermint/libs/os"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
 
-	// Osmosis application prarmeters
```

Comment:
Do you think we should revert these comments / the prior ordering of imports here?

Reply:
You mean keep them?

Import blocks aren't meant to be annotated. If you think devs would really find them useful, I can revert :)

Reply:
@faddat was the one who added them, presumably he's found it helpful when explaining app.go's to people.

We could put the same comments in the AppKeepers struct though, I don't think theres anything particularly important about this info being in the imports